styles.css

```diff
@@ -0,0 +1,768 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  line-height: 1.4;
+  max-width: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.toggler {
+  display: none;
+}
+
+.logo img {
+  max-width: 4rem;
+}
+
+.logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-right: 1.5rem;
+}
+
+.footer-logo-text {
+  font-size: 0.8rem;
+}
+
+.nav1 {
+  background-color: #212529;
+}
+
+.nav1-items {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0.5rem 4rem 0.5rem 0;
+}
+
+.nav1-link,
+.nav1-item {
+  margin: 0 1rem;
+}
+
+.nav2 {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 0 2rem 5rem;
+}
+
+.nav2-items {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  align-items: center;
+  margin-right: 5rem;
+}
+
+.nav2-link {
+  padding: 0 1em;
+  color: #668b9a;
+}
+
+.languages {
+  background-color: #212529;
+  color: #668b9a;
+  margin: 0 1rem;
+}
+
+.fa-facebook-f,
+.fa-twitter {
+  color: #fff;
+}
+
+.nav1-item {
+  color: #668b9a;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: end;
+}
+
+.menu-wrap {
+  background-color: #f8f9fa;
+}
+
+.about {
+  color: red;
+}
+
+.campaign {
+  color: red;
+  border: 2px solid red;
+  padding: 0.8rem 0.2rem;
+}
+
+.showcase {
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+  margin: auto;
+  padding: 0 3rem;
+}
```

Comment:
- [ ] This section looks fine. What we've got to work on are the margins/paddings. In the original design, this section's text aligns with the logo just above it. The screenshot below will help us realize the spacing and alignment issues that we still have to fix. Just compare and contrast to notice the difference and do the necessary adjustments.

| The Provided Design |
| --- |
| ![header-design](https://user-images.githubusercontent.com/45002654/133525014-d85756fd-14eb-4840-b014-46f09177de44.PNG) |

| The Implementation |
| --- |
| ![header-imp](https://user-images.githubusercontent.com/45002654/133524997-770c7c8f-4a37-42ce-a395-0d752cac30e9.PNG) |